taskcluster/mozillavpn_taskgraph/transforms/release_index.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from taskgraph.transforms.base import TransformSequence
+transforms = TransformSequence()
+
+
+@transforms.add
+def add_pr_number(config, tasks):
+   for task in tasks:
+        if task.__contains__("push-to-release-index"):
+            release_index = task.pop("push-to-release-index")
+            if release_index and int(config.params["level"]) == 3:
+                name = task['name'].split("/")[0]
+                version = config.params["app_version"]
+                route = f"index.mozillavpn.v2.mozillavpn.releases.{version}.{name}.latest"
```

Comment:
I guess it's a matter of taste, but I think the `latest` here is superfluous.

We may want to add a `index.mozillavpn.v2.mozillavpn.releases.latest.{name}` route, but then we'd need some way to check whether the current version is the highest or not. So maybe we can ignore that for now.